usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -2002,15 +2017,24 @@ static CK_RV import_RSA_key(STDLL_TokData_t * tokdata, SESSION * sess,
             TRACE_INFO("%s wrapping unwrap key rc=0x%lx blob_size=0x%zx\n",
                        __func__, rc, *blob_size);
         }
+
+        cleanse_attribute(rsa_key_obj->template, CKA_PRIVATE_EXPONENT);
+        cleanse_attribute(rsa_key_obj->template, CKA_PRIME_1);
+        cleanse_attribute(rsa_key_obj->template, CKA_PRIME_2);
+        cleanse_attribute(rsa_key_obj->template, CKA_EXPONENT_1);
+        cleanse_attribute(rsa_key_obj->template, CKA_EXPONENT_2);
+        cleanse_attribute(rsa_key_obj->template, CKA_COEFFICIENT);
     }
 
 import_RSA_key_end:
```

Comment:
exit pint after check_key_attributes should be mapped to this label, no ?

Reply:
fixed